### ISSUE TITLE
Usefulness of  statistical interpretation of DAGs

### ISSUE THREAD

**Issue by PRijnbeek** [mi-erasmusmc/CausalInference#1]
Question from Erica:
In the Coursera course there was discussion in the relation to probability.  In our field I see the use of the diagrams for understanding relationships however I don't see the direct use of the probability portion.  Could someone help me undersatnd how to think about this better?

**Comment by ericaVoss** [comment]
- discussion that writing out the probabilities might just help you understand the DAG
- probabilities can help you validate the DAG

**Comment by rossdwilliams** [comment]
So I was thinking a bit more about this and the connection with probabilities was nagging at me. I was wondering if Bayesian networks were built on top of DAGs. 

It seems to me that what a DAG does is set up the structure for the network based upon some knowledge of how the system we are considering works. 

In the case of looking for negative controls we have (taking the example Peter posted)

![image](https://user-images.githubusercontent.com/34747165/42881074-d2147e6c-8a96-11e8-9c9b-13aa30dbd086.png)

Which we know is the structure that this network has to take and then when we have data that fit into this system we can then start to work out the probabilities that are associated with each node. 

I'm not sure if this is a great example but it was just something I was wondering about

**Comment by rossdwilliams** [comment]
possibly something like this? https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3898602/pdf/nihms485650.pdf

**Comment by PRijnbeek** [comment]
Yes, there is a strong relationship between DAGs and belief networks.

the probabilistic interpretation of DAGs is discussed in the Coursera course:

https://www.coursera.org/learn/crash-course-in-causality/lecture/xihYv/relationship-between-dags-and-probability-distributions

**Comment by rossdwilliams** [comment]
Yeah so my thought from that is that given we have these belief networks can we use it to inform our priors? 

I'm wondering whether this could be useful for incorporating expert knowledge in terms of setting priors, although this is maybe off topic

**Comment by ericaVoss** [comment]
@PRijnbeek I think @rossdwilliams it touching on where I hope to get to with my PhD, using what we learn in the CommonEvidenceModel to inform our models.

**Comment by PRijnbeek** [comment]
@rossdwilliams yes absolutely Martijn has started with a implementation for informed priors we could work on together to move this forward. He is using sources like wikipedia for this. I will forward his presentation to you.
